classes/i18n/LocaleConversion.php

```diff
@@ -23,29 +23,76 @@
 
 namespace PKP\i18n;
 
+use Exception;
 use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Cache;
+use PKP\core\Core;
 use PKP\facades\Locale;
 
 class LocaleConversion
 {
+    /** @var string Max lifetime for the ISO639-2b cache. */
+    protected const MAX_ISO6392b_CACHE_LIFETIME = '1 year';
+
+    /**
+     * A post-install-cmd custom composer script that checks if
+     * the iso_639-2b.json file exists in the installed sokil library
+     *
+     * @throw Exception
+     */
+    public static function composerIsoFileCheckScript(): void
```

Comment:
As this isn't useful for the codebase, I think we should move it to a separated "build script", like "composer-commands.php".

Reply:
Yes, I was thinking about having a class ComposerScript or so, but I did not know where to put it -- I thought in lib/pkp/classes/composer/ -- and then I thought it currently only for this one small function, not even using any composer script functionality... Thus, I left it here for now...
Hmmm... I myself am not sure...
@jonasraoni or @asmecher, where would you put it?

Reply:
I added the class ComposerScript in lib/pkp/dev/ that contains this function...